### PR TITLE
Feat/mzad 44/reusable play button mobile

### PR DESCRIPTION
**Pull request by lizaBukia**

### CODE REVIEW

File: app/Components/PayButtonMobile/PlayButtonMobile.module.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@import '../../Colors/common-colors.scss';
+.playButton {
+    border: none;
```
Comment (pailodze) [inline app/Components/PayButtonMobile/PlayButtonMobile.module.scss:3]: remove this space

**Review by pailodze — CHANGES_REQUESTED** [review]

File: app/Components/Icon/enums/icon-name.enum.ts [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
   Album = 'album',
-  Backward = 'backward',
-  Forward = 'forward',
+  BackwardDesktop = 'backward-desktop',
+  BackwardLight = 'backward-light',
+  BackwardDark = 'backward-dark',
+  ForwardDesktop = 'forward-desktop',
+  ForwardLight = 'forward-light',
+  ForwardDark = 'forward-dark',
+  pauseIcon = "pauseIcon",
```
Comment (pailodze) [inline app/Components/Icon/enums/icon-name.enum.ts:25]: Names Of Icons Should Be In PascalCase

File: app/Components/PayButtonMobile/PlayButtonMobile.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  const onClink = () => {
+    setIsPlaying(!isPlaying);
+  };
+  return (
+    <button
+      onClick={() => {
+        onClink();
+        props.onClick;
+      }}
+      className={isPlaying ? `${styles.playButton} ${className} ${ styles.play}` : `${styles.playButton} ${className} ${ styles.notPlay}`}
```
Comment (pailodze) [inline app/Components/PayButtonMobile/PlayButtonMobile.tsx:28]: classNam={`${className} ${isPlaying ? styles.playButton : styles.notPlay}`}

File: app/Components/PayButtonMobile/PlayButtonMobile.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      style={{ width: props.width, height: props.width }}
+    >
+      <Icon name={icon} isActive={false} width={isPlaying ? 28 : 32} height={isPlaying ? 28 : 32} />
```
Comment (pailodze) [inline app/Components/PayButtonMobile/PlayButtonMobile.tsx:31]: You sure that this has to change with is playing?

File: app/Components/PayButtonMobile/enums/play-button-mobile-type.enum.ts [hunk 5]
```diff
@@ -0,0 +1,4 @@
+export enum PlayButtonMobileTypeEnum {
```
Comment (pailodze) [inline app/Components/PayButtonMobile/enums/play-button-mobile-type.enum.ts:1]: PlayButtonTypEnum

File: app/Components/PlayButton/interfaces/play-button-props.interface.ts [hunk 6]
```diff
@@ -0,0 +1,8 @@
+import { ReactNode } from 'react';
+
+export interface PlayButtonPropsInterface {
+  icon: string;
+  onClick: () => void;
```
Comment (pailodze) [inline app/Components/PlayButton/interfaces/play-button-props.interface.ts:5]: I think this is code duplicating use same interface in button mobile and button desktop

**Review by pailodze — CHANGES_REQUESTED** [review]

File: app/Components/Icon/enums/icon-name.enum.ts (hunk 2 above)
Comment (lizaBukia) [reply app/Components/Icon/enums/icon-name.enum.ts:25]: გვანცას როგორც ჰქონდა ისე ჩავამატე

File: app/Components/PayButtonMobile/PlayButtonMobile.tsx (hunk 4 above)
Comment (lizaBukia) [reply app/Components/PayButtonMobile/PlayButtonMobile.tsx:31]: yes i am sure

File: app/Components/PayButtonMobile/enums/play-button-mobile-type.enum.ts (hunk 5 above)
Comment (lizaBukia) [reply app/Components/PayButtonMobile/enums/play-button-mobile-type.enum.ts:1]: amas ver shevcvli eg saxeli aqvs prosta playbuttontypeenumsac

**Pull request merged**